lib/fuzion/std/fileio.fz

```diff
@@ -184,7 +184,7 @@ public fileio is
   public create_dir(
                 # the (relative or absolute) dir name, using platform specific path separators
                 path string) outcome unit is
-    arr := path.utf8.asArray
+    arr := path.utf8.asArray # NYI : #570
```

Comment:
please add a short comment if you reference an issue such that it is clear what the issue is (C string nul termination). 
#570 has been closed, so better update this patch to not introduce this problem again.